Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -214,6 +214,7 @@ public struct PackageBuilder {
         } else {
             testModules = []
         }
+        try fillDependencies(modules: modules + testModules)
```

Comment:
This should probably be done inside `constructTestModules` as it is for `constructModules`, and try to use the same code for resolving dependencies (if easy).

I think these two functions should be merged and refactored over time, FWIW...